packages/components/src/InputValidation/InputValidation.tsx

```diff
@@ -3,34 +3,15 @@ import { AnimatePresence, motion } from "framer-motion";
 import styles from "./InputValidation.css";
 import { Text } from "../Text";
 
-type ValidationStatus = "success" | "error" | "warn" | "info";
-
-export interface ValidationProps {
-  /**
-   * Defines the status of the validation message. This also determines how the
-   * message is styled in the UI.
-   */
-  status: ValidationStatus;
-
-  /**
-   * The message that gets shown to the user.
-   */
-  message: string;
-
-  /**
-   * Determines if the `message` shows up for the user.
-   */
-  shouldShow?: boolean;
-}
-
 interface InputValidationProps {
   /**
-   * Array of validation messages
+   * Validation message to be displayed
    */
-  messages: ValidationProps[];
+  message: string;
```

Comment:
If `onValidation` can call with `undefined` I wonder if this should also accept `undefined`?

Reply:
This should always accept a string. The other undefined was not needed.